packages/rspack-dev-server/src/index.ts

```diff
@@ -1,2 +1,7 @@
+import pkgJson from "@rspack/dev-server/package.json";
+
+const { version: rspackDevServerVersion } = pkgJson;
+export { rspackDevServerVersion };
```

Comment:
We can export `version` instead of `rspackDevServerVersion`

```suggestion
export { version };
```

Reply:
I think there is no difference between `import { version } from '@rspack/dev-server'` and `import pkgJson from '@rspack/dev-server/package.json'`.

If we need to support `this.RspackDS.version`, maybe we need add the version to somewhere else?